encrypted-media-respec.html

```diff
@@ -3218,6 +3218,23 @@
             <li><p>MediaKeys objects and the underlying implementation do not expose information outside the <a def-id="origin"></a>.</p></li>
             <li><p>Persisted session data, if applicable, is stored on a per-<a def-id="origin"></a> basis.</p></li>
             <li><p>Only data stored by the requesting <a def-id="origin"></a> may be loaded.</p></li>
+            <li>
+              <p>
+                It is not possible to extract, derive or infer information from CDM messages that is not either explicitly described in this specification
```

Comment:
Rather than saying "CDM messages," which could be interepreted as `message` events, we might want to use language similar to Distinctive Identifier: "It is exposed outside the client device or exposed to the application..."